packages/trader/src/AppV2/Components/DealCancellationRemainingTime/deal-cancellation-remaining-time.tsx

```diff
@@ -0,0 +1,37 @@
+import { Tag } from '@deriv-com/quill-ui';
+import { LabelPairedStopwatchCaptionRegularIcon } from '@deriv/quill-icons';
+import { formatDuration, getDiffDuration } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import React from 'react';
+
+type DealCancellationRemainingTimeProps = {
+    format?: string;
+};
+
+const DealCancellationRemainingTime = observer(({ format = 'mm:ss' }: DealCancellationRemainingTimeProps) => {
+    const { common } = useStore();
+    const { server_time: start_time } = common;
+    const { contract_info } = useContractDetails();
+
+    const { date_expiry: end_time } = contract_info.cancellation || {};
+    if (!end_time || start_time.unix() > +end_time) {
+        return <React.Fragment>{''}</React.Fragment>;
```

Comment:
Do we need to return empty string? Maybe `null` or create one conditional rendering?

Reply:
Same thing honestly, but I'll change